UKHO.ExchangeSetService.API/UKHO.ExchangeSetService.Common/Logging/EventIds.cs

```diff
@@ -512,7 +512,12 @@ public enum EventIds
         /// <summary>
         /// 805126 - Completed query for ENC files based on Product/CellName EditionNumber and UpdateNumber from file share service.
         /// </summary>
-        FileShareServiceSearchQueryForSetOfENCsCompleted = 805126
+        FileShareServiceSearchQueryForSetOfENCsCompleted = 805126,
+        /// <summary>
+        /// 805127 - Cancellation tokken called when any of the async task in Parllel thread is failed
+        /// </summary>
+        CancellationTokenEvent = 805127
```

Comment:
Can we change summary to Cancellation of task/token is called/requested when any of the async task in parallel thread is failed.